pandas/tests/series/test_period.py

```diff
@@ -71,10 +71,9 @@ def test_NaT_scalar(self):
         series[2] = val
         assert pd.isna(series[2])
 
-    @pytest.mark.xfail(reason="PeriodDtype Series not supported yet")
     def test_NaT_cast(self):
         result = Series([np.nan]).astype("period[D]")
-        expected = Series([pd.NaT])
+        expected = Series([pd.NaT], dtype="period[D]")
```

Comment:
This is changing a test, no?  Why are we dropping the `xfail` ?

Reply:
IIUC the xfail was from a time before `dtype="period[D]"` was recognized